WordPress/Classes/Services/ReaderPostService.m

```diff
@@ -1015,7 +1010,16 @@ - (ReaderPost *)createOrReplaceFromRemotePost:(RemoteReaderPost *)remotePost for
     post.score = remotePost.score;
     post.siteID = remotePost.siteID;
     post.sortDate = remotePost.sortDate;
-    post.sortRank = remotePost.sortRank;
+
+    if (!existing) {
+        // Failsafe.  The `read/searc` endpoint might return the same post on
+        // more than one page. If this happens preserve the *original* sortRank
+        // to avoid content jumping around in the UI.
+        // Posts from other endpoits will store a date value which shouldn't
```

Comment:
Can I just blame my keyboard keys?  They're broken.  Yeah.  That's what happened.